README.rst

```diff
@@ -582,19 +639,60 @@ Response Body
 The `statement`_ object.
 
 
+GET /sessions/{sessionId}/statements/{statementId}
+--------------------------------------------------
+
+Returns a specified statement in a session.
+
+Response Body
+^^^^^^^^^^^^^
+
+The `statement`_ object.
+
+
+POST /sessions/{sessionId}/statements/{statementId}/cancel
+----------------------------------------------------------
+
+Cancel the specified statement in this session.
+
+Response Body
+^^^^^^^^^^^^^
+
++------+----------------------------+--------+
+| name | description                | type   |
++======+============================+========+
+| msg  | denote cancelled statement | string |
++------+----------------------------+--------+
+
+
 GET /batches
-------------
+-------------
+
+Returns all the active batch sessions.
 
-Returns all the active batch jobs.
+Request Parameters
+^^^^^^^^^^^^^^^^^^
+
++------+-----------------------------------+------+
+| name | description                       | type |
++======+===================================+======+
+| from | The start session id to fetch     | int  |
```

Comment:
I don't think this's right. It's not a session id. It's the the index of the first shown session id in SessionServlet.sessions array.

However, I think what you are suggesting makes sense. We should change the code to match this behavior.